package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "live-coding",
```

Comment:
This is not live coding. :) Live coding session is a session that you share your screen and do coding.